pkgs/tools/admin/simp_le/default.nix

```
@@ -2,11 +2,11 @@
 
 python3Packages.buildPythonApplication rec {
   pname = "simp_le-client";
-  version = "0.17.0";
+  version = "0.20.0";
```

```suggestion
  version = "0.20.0";
  pyproject = true;
```

To use the PEP517 builder. Then add `setuptools` into `nativeBuildInputs`.